crawl-ref/source/mon-cast.cc

```diff
@@ -3503,7 +3503,7 @@ monster* cast_phantom_mirror(monster* mons, monster* targ, int hp_perc,
 
     // Sometimes swap the two monsters, so as to disguise the original and the
     // copy.
-    if (coinflip())
+    if (x_chance_in_y(1, swap_chance_reciprocal))
```

Comment:
You can use `one_chance_in` here. (Same thing, just a little shorter and nicer to read.)